fusor-ember-cli/app/routes/rhev-options.js

```diff
@@ -1,6 +1,12 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  setupController(controller, model) {
+    controller.set('model', model);
+    controller.set('confirm_rhev_root_password', this.controllerFor('deployment').get('model.rhev_root_password'));
+    controller.set('confirm_rhev_engine_admin_password', this.controllerFor('deployment').get('model.rhev_engine_admin_password'));
```

Comment:
I suggest to use camelCase here. The snake_case usually refers to a model attribute which these are not.